copilot/fsd-authenticator/manifest.yml

```diff
@@ -49,8 +49,8 @@ variables:
   FUND_STORE_API_HOST: "http://fsd-fund-store:8080"
   ACCOUNT_STORE_API_HOST: "http://fsd-account-store:8080"
   AUTHENTICATOR_HOST: "https://authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
-  POST_AWARD_FRONTEND_HOST: "https://find-monitoring-data.${data.cloudfoundry_domain.workspace.name}"
-  POST_AWARD_SUBMIT_HOST: "https://submit-monitoring-data.${data.cloudfoundry_domain.workspace.name}"
+  POST_AWARD_FRONTEND_HOST: "https://find-monitoring-data.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
```

Comment:
Can we leave POST_AWARD_FRONTEND_HOST as-is for now until we've fully switched that app over to the new domain?